pkg/katautils/create.go

```diff
@@ -161,9 +161,12 @@ func HandleFactory(ctx context.Context, vci vc.VC, runtimeConfig *oci.RuntimeCon
 // of the same pod the already existing volume is reused.
 func SetEphemeralStorageType(ociSpec oci.CompatOCISpec) oci.CompatOCISpec {
 	for idx, mnt := range ociSpec.Mounts {
-		if IsEphemeralStorage(mnt.Source) {
+		if vc.IsEphemeralStorage(mnt.Source) {
 			ociSpec.Mounts[idx].Type = "ephemeral"
 		}
+		if vc.Isk8sHostEmptyDir(mnt.Source) {
+			ociSpec.Mounts[idx].Type = "local"
```

Comment:
is it better to replace "local" with vc.kataLocalDevType ?

Reply:
Yea it makes sense to do that, unfortunately `vc.kataLocalDevType` & `vc.kataEphemeralDevType` are private, should I make them public?

Reply:
Yes,  please.